src/protocols/aeternity/views/TransactionDetails.vue

```diff
@@ -319,10 +320,11 @@ export default defineComponent({
     onMounted(async () => {
       let rawTransaction = getTransactionByHash(activeAccount.value.address, hash);
 
-      if (!rawTransaction || rawTransaction.incomplete) {
+      // Claim transactions have missing data that needs to be fetched from the middleware
+      if (!rawTransaction || rawTransaction.incomplete || rawTransaction.claim) {
         const middleware = await getMiddleware();
         try {
-          rawTransaction = await middleware.getTx(hash);
+          rawTransaction = merge(rawTransaction, await middleware.getTx(hash));
```

Comment:
Why this `merge` is needed?

Reply:
Because in the case of `claim` transactions we want to merge data coming from the backend & the middleware so that we always have the same (correct and complete) transaction.